src/device.rs

```diff
@@ -0,0 +1,91 @@
+use crate::AxVmDeviceConfig;
```

Comment:
`use alloc::*` or `use core::*` should be placed in front of others.

Reply:
After running cargo fmt, it becomes like this

Reply:
add blank lines between them